changelog/_unreleased/2025-09-25-implement-validate-agnostic-filename-command.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Implement validate agnostic filename command
+issue: #12188
+author: Marcel Brode
+author_email: m.brode@shopware.com
+author_github: @Marcel Brode
+---
+# Core
+* Added a new CLI command `translation:validate-filenames` to validate filenames against agnostic naming conventions
+* Changed command `snippet:validate` to `translation:validate` to scope all translation related commands under one namespace
```

Comment:
is renaming a command a breaking change?

Reply:
It is! Good catch. Maybe external scripts, cronjobs or ci/cd pipelines are working with them. We should provide `translation:validate` as an own Command and deprecate the old one.